lib/events/base.rb

```diff
@@ -9,6 +9,6 @@ def initialize(name, timestamp, data)
   end
 
   def method_name
-    name.to_s.gsub('.', '_')
+    name.to_s.tr('.', '_')
```

Comment:
Any reason why this is needed?

Reply:
its an automatic rubocop substitution for better performance 

```
gsub is used for replacing substrings with new substrings, while tr is used for replacing individual characters with other characters.
```